### PR TITLE
[CSL-2242] Add background images to select questions

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: src/components/CioQuiz/index.tsx [hunk 1]
```diff
@@ -80,6 +88,9 @@ export default function CioQuiz(props: IQuizProps) {
         <QuizContext.Provider value={contextValue}>
           {state.quiz.results && <ResultContainer options={resultsPageOptions} />}
           {state.quiz.currentQuestion && <QuizQuestions />}
+          {state.quiz.currentQuestion && (
+            <ControlBar ctaButtonText={questionData?.cta_text || undefined} />
```
Comment (esezen) [inline src/components/CioQuiz/index.tsx:92]: Typescript wasn't happy with `questionData?.cta_text` here, let me know if there is a better way to do this

**Review by stanlp1 — APPROVED** [review]
LGTM. Everything appears to work as expected.

File: src/components/CioQuiz/index.tsx [hunk 2]
```diff
@@ -87,7 +95,12 @@ export default function CioQuiz(props: IQuizProps) {
           {state.quiz.results || state.quiz.skipToResults ? (
             <ResultContainer options={resultsPageOptions} />
           ) : (
-            state.quiz.currentQuestion && <QuizQuestions />
+            state.quiz.currentQuestion && (
+              <>
+                <QuizQuestions />
+                <ControlBar ctaButtonText={questionData?.cta_text || undefined} />
```
Comment (deomsj) [inline src/components/CioQuiz/index.tsx:101]: `questionData?.cta_text || undefined`

could be simplified to: `questionData?.cta_text`

right?

File: src/styles.css [hunk 3]
```diff
@@ -86,7 +92,15 @@
   padding-bottom: var(--bottom-bar-height);
   position: relative;
   align-self: start;
-  min-height: 100%;
+  height: calc(100% - calc(var(--bottom-bar-height))); /* Full height - margin - control bar */
```
Comment (deomsj) [inline src/styles.css:95]: css math. nice

File: src/styles.css [hunk 4]
```diff
@@ -302,12 +318,15 @@
...
   flex-direction: column;
   align-items: center;
-  align-self: start;
   max-width: 1200px;
   padding: 1rem;
   margin-top: 2rem;
-  min-height: calc(100% - 3rem); /* Full height - padding - margin */
+  min-height: calc(
+    100% - 2rem - calc(var(--bottom-bar-height))
+  ); /* Full height - padding - margin */
```
Comment (deomsj) [inline src/styles.css:326]: can we simplify:
```
calc(
    100% - 2rem - calc(var(--bottom-bar-height))
  )
```
  
to: `calc(100% - 2rem - var(--bottom-bar-height))`

File: src/styles.css [hunk 5]
```diff
@@ -623,6 +644,7 @@
   .cio-container {
     align-items: center;
     margin-top: 0;
+    height: calc(100% - calc(var(--bottom-bar-height))); /* Full height - control bar */
```
Comment (deomsj) [inline src/styles.css:647]: same as above, do we need two different calc functions in this expression?

File: src/styles.css [hunk 6]
```diff
@@ -686,9 +698,11 @@
   }
   .cio-select-question-container {
     justify-content: center;
-    margin: 0;
+    margin-bottom: 0;
+    margin-top: 0;
     padding: 2rem;
     padding-bottom: 0;
+    min-height: calc(100% - calc(var(--bottom-bar-height))); /* Full height - padding - margin */
```
Comment (deomsj) [inline src/styles.css:705]: same as above

**Review by deomsj — COMMENTED** [review]
nice. just a few comments on potential opportunities to simplify code

File: src/components/CioQuiz/index.tsx (hunk 2 above)
Comment (esezen) [reply src/components/CioQuiz/index.tsx:101]: Typescript gets really mad because that returns null and null is not undefined

**Pull request merged**